sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-eCert-errors-report.sql

```diff
@@ -0,0 +1,49 @@
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'ECert_Errors_Report',
+        'select
+            disbursement_schedules.id as "eCert Number",
```

Comment:
I believe the `eCert Number` should be the e-Cert document number, please check.